### PR TITLE
Indicate score properties are inclusive

### PR DESCRIPTION
**Pull request by brianjmiller**

### CODE REVIEW

File: cmi5_spec.md [hunk 1]
```diff
@@ -1045,7 +1045,7 @@ The properties for the "LMS.LaunchData" document are described below.
   <tr><th align="right" nowrap>LMS Usage:</th><td>The LMS MUST include the "masteryScore" value based on the value defined in the course structure for the AU being launched.</td></tr>
   <tr><th align="right" nowrap>AU Usage:</th><td>The AU MUST issue Passed or Failed statements based on the "masteryScore" provided. (See Sections 9.3.6 and 9.3.7)</td></tr>
   <tr><th align="right" nowrap>Data Type:</th><td>decimal</td></tr>
-  <tr><th align="right" nowrap>Value Space:</th><td>Decimal value between 0 and 1.</td></tr>
+  <tr><th align="right" nowrap>Value Space:</th><td>Decimal value between 0 and 1 (inclusive).</td></tr>
```
Comment (brianjmiller) [inline cmi5_spec.md]: I'm no longer sure this makes sense at least in one half, can a mastery score be 0? Perhaps it needs to be `greater than 0 and less than or equal to 1`?

File: cmi5_spec.md [hunk 2]
```diff
@@ -1425,7 +1425,7 @@ The data in this section are used by the LMS to locate the AU and provide launch
       <ul>
         <li>The "masteryScore" is passed to the AU at runtime by the LMS (as defined in the cmi5 Runtime Specification).</li>
         <li>If the AU has scoring, it will use the "masteryScore" to determine pass/fail (as defined in the cmi5 Runtime Specification)</li>
-        <li>The "masteryScore" is a scaled, decimal value between 0 and 1.</li>
+        <li>The "masteryScore" is a scaled, decimal value between 0 and 1 (inclusive).</li>
```
Comment (brianjmiller) [inline cmi5_spec.md]: See above comment.

**Comment by brianjmiller** [comment]
It was indicated on the 12/4 call that masteryScore should be allowed to contain 0.

**Comment by cawerkenthin** [comment]
:+1:

**Pull request merged**